RUFAS/routines/manure/gas_emissions/gas_emissions.py

```diff
@@ -647,3 +648,177 @@ def calc_methane_emission_from_anaerobic_lagoon(cls, manure_volatile_solids: flo
         MS = GasEmissionConstants.FRACTION_OF_HANDLED_MANURE
         MF = GasEmissionConstants.METHANE_FACTOR
         return Bo * MCF * MS * MF * manure_volatile_solids
+
+    @classmethod
+    def calc_mcf(cls, ambient_barn_temp: float) -> float:
+        """
+        Calculate the Methane Conversion Factor (MCF) using the exponential function:
+        MCF(T) = MCF_CONSTANT_A * e^(MCF_CONSTANT_B * T)
+
+        Parameters
+        ----------
+        ambient_barn_temperature : float
+            The ambient barn temperature (in Celsius).
+
+        Returns
+        -------
+        float
+            The calculated Methane Conversion Factor (MCF) for the given ambient barn temperature.
+
+        """
+        return GasEmissionConstants.MCF_CONSTANT_A * math.exp(GasEmissionConstants.MCF_CONSTANT_B * ambient_barn_temp)
+
+    @classmethod
+    def calc_ifsm_methane_emission(cls, manure_volatile_solids: float, ambient_barn_temp: float) -> float:
+        """Calculates emission of methane for a day using an adaptation of the tier 2 approach
+        of the IPCC(2006), given ambient barn temperature and a methane conversion factor for the manure
+        management.
+
+        Parameters
+        ----------
+        manure_volatile_solids : float
+            The volatile solids (in kg)
+        ambient_barn_temperature : float
+            The ambient barn temperature (in Celsius)
```

Comment:
```suggestion
        manure_volatile_solids : float
            The volatile solids (in kg)
            
        ambient_barn_temperature : float
            The ambient barn temperature (in Celsius)
```
Consistent docstring formatting?